white-test-app-frontend/src/app/test-templates/modify-test-modal.html

```diff
@@ -1,12 +1,15 @@
-<div class="modal-header">
-  <h4 class="modal-title" id="modal-basic-title">Dodaj test</h4>
-  <button type="button" class="close" aria-label="Close" (click)="activeModal.dismiss('Cross click')">
-    <span aria-hidden="true">&times;</span>
-  </button>
-</div>
+<div (mouseup)="getSelectedText(); mouseClick()()" (window:keydown.shift.arrowup)="openWikipedia($event)"
+     (window:keydown.shift.arrowdown)="findSynonyms($event)">
+  <div class="modal-header">
+    <h4 class="modal-title" id="modal-basic-title">Dodaj test</h4>
+    <button type="button" class="close" aria-label="Close" (click)="activeModal.dismiss('Cross click')">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
 
-<app-questions *ngIf="testTemplate" [questions]="testTemplate.questions"></app-questions>
+  <app-questions *ngIf="testTemplate" [questions]="testTemplate.questions"></app-questions>
```

Comment:
spacja niepotrzebna

Reply:
potrzebna, bo div został dodany i się tak sformatowało :P

Reply:
A okej, rozumiem to resolve